pkg/testcase/secretsencrypt.go

```diff
@@ -22,84 +24,91 @@ func TestSecretsEncryption() {
 	errSecret := shared.CreateSecret("secret1", "default")
 	Expect(errSecret).NotTo(HaveOccurred(), "error creating secret")
 
-	shared.LogLevel("info", "TEST: 'CLASSIC' Secrets Encryption method")
-
 	index := len(nodes) - 1
 	cpIp := nodes[index].ExternalIP
-	secretsEncryptOps("prepare", product, cpIp, nodes)
-	secretsEncryptOps("rotate", product, cpIp, nodes)
-	secretsEncryptOps("reencrypt", product, cpIp, nodes)
 
-	if strings.Contains(os.Getenv("TEST_TYPE"), "both") {
-		shared.LogLevel("info", "TEST: 'NEW' Secrets Encryption method")
-		secretsEncryptOps("rotate-keys", product, cpIp, nodes)
-	}
+	secretsEncryptOps("prepare", product, cluster.ServerIPs[0], cpIp, nodes)
```

Comment:
Can we add debug logging here before and after each action similar to https://github.com/rancher/distros-test-framework/pull/194/files#r1882119964

Reply:
The log line is inside the secretsEncryptOps method: https://github.com/rancher/distros-test-framework/blob/main/pkg/testcase/secretsencrypt.go#L40
Added a debug log line at the end of the method action step completed as well.